File: API/Entities/Event.cs
namespace API.Entities
{
    public class Event
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Purpose { get; set; }
        public string EventCode { get; set; }
        public string TeamInCharge { get; set; }
        public DateTime PostingDate { get; set; }
        public DateTime DraftDeadline { get; set; }
        public string Specifications { get; set; }
        public ICollection<EventPhoto> Photos { get; set; }

    }
}

File: client/src/app/_models/event.ts
import { EventPhoto } from "./eventPhoto";

export interface ClubEvent {
    id: number;
    title: string;
    photoUrl: string;
    purpose: string;
    teamInCharge: string;
    postingDate: Date;
    draftDeadline: Date;
    specifications: string;
    photos: EventPhoto[];
}